frontend/src/containers/data-tool/sidebar/widgets/establishment-stages/index.tsx

```diff
@@ -4,14 +4,20 @@ import { groupBy } from 'lodash-es';
 
 import HorizontalBarChart from '@/components/charts/horizontal-bar-chart';
 import Widget from '@/components/widget';
-import { ESTABLISHMENT_STAGES_CHART_BAR_BACKGROUNDS } from '@/constants/establishment-stages-chart-bar-backgrounds';
 import { useGetMpaaEstablishmentStageStats } from '@/types/generated/mpaa-establishment-stage-stat';
 import type { LocationGroupsDataItemAttributes } from '@/types/generated/strapi.schemas';
 
 type EstablishmentStagesWidgetProps = {
   location: LocationGroupsDataItemAttributes;
 };
 
+const PATTERNS = {
+  'proposed-committed': '/images/data-tool/chart-bgs/dots.svg',
+  implemented: '/images/data-tool/chart-bgs/crosses.svg',
+  'actively-managed': '/images/data-tool/chart-bgs/dashes.svg',
+  designated: '/images/data-tool/chart-bgs/arrows.svg',
+};
+
```

Comment:
@andresgnlez Strapi has a couple more defined [here](https://30x30.skytruth.org/cms/admin/content-manager/collectionType/api::mpaa-establishment-stage.mpaa-establishment-stage?page=1&pageSize=10&sort=slug:ASC) ,we may want to add them as well in case they get used (`designated-unimplemented` and `unknown`)

Reply:
@SARodrigues just talked to Sofía, and she will delete the ones unused. We should be fine with these ones.